definitions/infra-kubernetes_daemonset/definition.yml

```diff
@@ -0,0 +1,31 @@
+domain: INFRA
+type: KUBERNETES_DAEMONSET
+configuration:
+  entityExpirationTime: DAILY
+  alertable: true
+
+synthesis:
+  identifier: entity.id
+  name: k8s.daemonsetName
+  encodeIdentifierInGUID: false
```

Comment:
I'm wondering if things that come from a user-generated name should be always encoded.
My thinking is that, same as hostname, we don't control lengths and it could be over the limit.

@srvaroa any input on this one?

Reply:
If no dashboard is provided users who have this entity will see an error message on the entity overview @douglascamata